ansible/vars/topo_t1-u2d254.yaml

```diff
@@ -0,0 +1,5650 @@
+      vm_offset: 228
+    ARISTA228T0:
+      vlans:
+        - 229
+      vm_offset: 229
+    ARISTA229T0:
+      vlans:
+        - 230
+      vm_offset: 230
+    ARISTA230T0:
+      vlans:
+        - 231
+      vm_offset: 231
+    ARISTA231T0:
+      vlans:
+        - 232
+      vm_offset: 232
+    ARISTA232T0:
+      vlans:
+        - 233
+      vm_offset: 233
+    ARISTA233T0:
+      vlans:
+        - 234
+      vm_offset: 234
+    ARISTA234T0:
+      vlans:
+        - 235
+      vm_offset: 235
+    ARISTA235T0:
+      vlans:
+        - 236
+      vm_offset: 236
+    ARISTA236T0:
+      vlans:
+        - 237
+      vm_offset: 237
+    ARISTA237T0:
+      vlans:
+        - 238
+      vm_offset: 238
+    ARISTA238T0:
+      vlans:
+        - 239
+      vm_offset: 239
+    ARISTA239T0:
+      vlans:
+        - 240
+      vm_offset: 240
+    ARISTA240T0:
+      vlans:
+        - 241
+      vm_offset: 241
+    ARISTA241T0:
+      vlans:
+        - 242
+      vm_offset: 242
+    ARISTA242T0:
+      vlans:
+        - 243
+      vm_offset: 243
+    ARISTA243T0:
+      vlans:
+        - 244
+      vm_offset: 244
+    ARISTA244T0:
+      vlans:
+        - 245
+      vm_offset: 245
+    ARISTA245T0:
+      vlans:
+        - 246
+      vm_offset: 246
+    ARISTA246T0:
+      vlans:
+        - 247
+      vm_offset: 247
+    ARISTA247T0:
+      vlans:
+        - 248
+      vm_offset: 248
+    ARISTA248T0:
+      vlans:
+        - 249
+      vm_offset: 249
+    ARISTA249T0:
+      vlans:
+        - 250
+      vm_offset: 250
+    ARISTA250T0:
+      vlans:
+        - 251
+      vm_offset: 251
+    ARISTA251T0:
+      vlans:
+        - 252
+      vm_offset: 252
+    ARISTA252T0:
+      vlans:
+        - 253
+      vm_offset: 253
+    ARISTA253T0:
+      vlans:
+        - 254
+      vm_offset: 254
+    ARISTA254T0:
+      vlans:
+        - 255
+      vm_offset: 255
+
+configuration_properties:
+  common:
+    dut_asn: 4200100000
+    dut_type: LeafRouter
+    podset_number: 200
+    tor_number: 16
+    tor_subnet_number: 2
+    max_tor_subnet_number: 16
+    tor_subnet_size: 128
+    nhipv6: FC0A::FF
+  spine:
+    swrole: spine
+  tor:
+    swrole: tor
+
+configuration:
+  ARISTA01T2:
+    properties:
+    - common
+    - spine
+    bgp:
+      router-id: 0.12.0.1
+      asn: 4200200000
+      peers:
+        4200100000:
+          - fc00:a::1
+    interfaces:
+      Loopback0:
+        ipv6: fc00:c:c::1/64
+      Ethernet1:
+        ipv6: fc00:a::2/126
+    bp_interfaces:
+      ipv6: fc00:b::1/64
+
+  ARISTA02T2:
+    properties:
+    - common
+    - spine
+    bgp:
+      router-id: 0.12.0.2
+      asn: 4200200000
+      peers:
+        4200100000:
+          - fc00:a::3
+    interfaces:
+      Loopback0:
+        ipv6: fc00:c:c::2/64
+      Ethernet1:
+        ipv6: fc00:a::4/126
+    bp_interfaces:
+      ipv6: fc00:b::2/64
+
+  ARISTA01T0:
+    properties:
+    - common
+    - tor
+    bgp:
+      router-id: 0.12.0.3
+      asn: 4200000000
```

Comment:
downlink asn need to be different.

Reply:
Will update asn after policy was confirmed.